operator/main.go

```diff
@@ -80,6 +86,10 @@ type ConfigSpec struct {
 	CertDir              string   `yaml:"cert-dir" jsonschema:"required"`
 	CertName             string   `yaml:"cert-name" jsonschema:"required"`
 	KeyName              string   `yaml:"key-name" jsonschema:"required"`
+	Log4jConfigName      string   `yaml:"log4j-config-name" jsonschema:"required"`
+
+	EaseagentInitializerImageName string `yaml:"easeagent-initializer-image-name" jsonschema:"required"`
```

Comment:
Can we use a general name such as `AgentInitializerImageName `, from a product view, we should not bind with EaseAgent.